### DISCUSSION TITLE
How to set multiple color based on recent modify date

### DISCUSSION THREAD

**Discussion by Jimmy-JS** [eza-community/eza#1237] [Q&A, Howto, Help]
How to set multiple colors based on recently modified date just like in the readme of eza themes (https://github.com/eza-community/eza-themes)?
![image](https://github.com/user-attachments/assets/88ddd35e-6ca7-48d5-bed0-5bdad939b9e4)

I want to make the more recently modified file to be lighter in color and darker for the older one. Since I don't see this in the theme yml

**Comment by leandrocm86** [comment]
> How to set multiple colors based on recently modified date just like in the readme of eza themes (https://github.com/eza-community/eza-themes)?
> ![image](https://github.com/user-attachments/assets/88ddd35e-6ca7-48d5-bed0-5bdad939b9e4)
> 
> I want to make the more recently modified file to be lighter in color and darker for the older one. Since I don't see this in the theme yml

I second this! And I don't understand why the date colors look the way they are in the examples. Here they all look the same.
